test/e2e/tests/onboarding.spec.js

```diff
@@ -19,6 +28,188 @@ describe('MetaMask onboarding', function () {
+          testSeedPhrase,
+          testPassword,
+        );
+      },
+    );
+  });
+
+  it('Clicks import a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeImportSRPOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
+      },
+    );
+  });
+
+  it('User import wrong secure password', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await importWrongSRPOnboardingFlow(driver, wrongSeedPhrase);
+      },
+    );
+  });
+
+  it('Check if user select different type your secret recovery phrase', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        // welcome
+        await driver.clickElement('[data-testid="onboarding-import-wallet"]');
+
+        await driver.clickElement('[data-testid="metametrics-no-thanks"]');
+
+        const dropdowns = await driver.findElements('select');
+        const dropdownElement = dropdowns[1];
+        await dropdownElement.click();
+        const options = await dropdownElement.findElements(
+          By.tagName('option'),
+        );
+
+        const iterations = options.length;
+
+        await testDropdownIterations(options, driver, iterations);
+      },
+    );
+  });
+
+  it('User enters the wrong password during password creation', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await driver.clickElement('[data-testid="onboarding-create-wallet"]');
+
+        // metrics
+        await driver.clickElement('[data-testid="metametrics-no-thanks"]');
+
+        // Fill in confirm password field with incorrect password
+        await driver.fill('[data-testid="create-password-new"]', testPassword);
+        await driver.fill(
+          '[data-testid="create-password-confirm"]',
+          wrongTestPassword,
+        );
+
+        // Check that the error message is displayed for the password fields
+        const errorMessages = await driver.findElements('h6');
```

Comment:
We could be more specific in our selector search, so instead of looking for all `h6` and then grabbing the 4th, we could look for the specific error text as well, so we can skip 1 step:

`await driver.findElement({ text: 'Settings', tag: 'div' })`

Reply:
Done